src/sqs/sqs.types.ts

```diff
@@ -138,6 +138,7 @@ export type BranchMessagePayload = BaseMessagePayload & {
 export type BackfillMessagePayload = BaseMessagePayload & {
 	syncType?: SyncType,
 	startTime?: string,
+	syncRequestedTime?: string, //ISO format
```

Comment:
Don't want to use `startTime` as it is being used purely for `full` sync only. Afraid it might mess up the existing sfx dashboard.